components/idp-mgt/org.wso2.carbon.idp.mgt.ui/src/main/java/org/wso2/carbon/idp/mgt/ui/util/IdPManagementUIUtil.java

```diff
@@ -1311,6 +1311,15 @@ private static void buildOpenIDConnectAuthenticationConfiguration(IdentityProvid
         }
         properties[8] = property;
 
+        property = new Property();
+        property.setName(IdentityApplicationConstants.Authenticator.OIDC.USE_OIDC_CLAIM_DIALECT);
+        if ("on".equals(paramMap.get(IdentityApplicationConstants.Authenticator.OIDC.USE_OIDC_CLAIM_DIALECT))) {
```

Comment:
Better to have a constant for "on"

Reply:
Can't we directly set the condition as the value to the property? Hope if-else condition is not necessary here

Reply:
Added as a constant. This is a checkbox and HTML forms don't submit a property for checkboxes when they are not checked. That's why we need the if-else condition.